### ISSUE TITLE
gophertunnel sometimes provides invalid checksums

### ISSUE THREAD

**Issue by TwistedAsylumMC** [Sandertv/gophertunnel#45]
When connecting to a PocketMine-MP v4.0 server, there is the chance that gophertunnel gives this error:
```
error verifying packet: invalid packet checksum: x should be y
```
Where x is the checksum from gophertunnel, and y is the checksum PocketMine expects

**Comment by Sandertv** [comment]
This might be fixed in the latest go-raknet update.

**Comment by Flonja** [comment]
This is now fixed, right? Or should it be open until we don't get anymore?

**Comment by Sandertv** [comment]
I think there is still an (unrelated) issue where this message shows up when closing the connection. It doesn't really impact anything, but I still do want to investigate that if it still exists.

**Comment by Sandertv** [comment]
Fixed in 0806af90a923fff79a8e291ed90b37e972f23712.
